--- config.def.h	2020-12-30 17:49:51.019795519 +0530
+++ config.def.h	2020-12-30 17:54:41.366760354 +0530
@@ -1,6 +1,6 @@
 /* user and group to drop privileges to */
 static const char *user  = "nobody";
-static const char *group = "nogroup";
+static const char *group = "nobody";
 
 static const char *colorname[NUMCOLS] = {
 	[INIT] =   "black",     /* after initialization */
@@ -13,16 +13,16 @@
 static const int failonclear = 1;
 
 /* time in seconds to cancel lock with mouse movement */
-static const int timetocancel = 4;
+static const int timetocancel = 5;
 
 /* default message */
-static const char * message = "Suckless: Software that sucks less.";
+static const char * message = "Enter password to unlock.";
 
 /* text color */
 static const char * text_color = "#ffffff";
 
 /* text size (must be a valid size) */
-static const char * font_name = "6x10";
+static const char * font_name = "-xos4-terminus-bold-r-normal--32-320-72-72-c-160-iso10646-1";
 
 /* time in seconds before the monitor shuts down */
 static const int monitortime = 5;
